src/main/java/com/karankumar/bookproject/ui/goal/SaveGoalEvent.java

```diff
@@ -0,0 +1,11 @@
+package com.karankumar.bookproject.ui.goal;
+
+import com.karankumar.bookproject.backend.entity.ReadingGoal;
+
+public class SaveGoalEvent extends ComponentGoalEvent.GoalFormEvent{
+
+        SaveGoalEvent(ReadingGoalForm source, ReadingGoal readingGoal) {
+            super(source, readingGoal);
+        }
+    }
```

Comment:
This last curly brace is incorrectly indented. It should be four spaces to the left